rules/no-unreadable-array-destructuring.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const getDocsUrl = require('./utils/get-docs-url');
+
+const isCommaFollwedWithComma = (el, index, array) => {
+	if (el === null) {
+		return array[index + 1] === el;
+	}
+};
+
+const create = context => {
+	return {
+		ArrayPattern(node) {
+			const {elements} = node;
+			if (!elements || elements.length === 0) {
+				return;
+			}
+
+			if (elements.some(isCommaFollwedWithComma)) {
```

Comment:
I think you would get the same result, but faster if you changed this condition to `elements.find(isCommaFollwedWithComma)`

Reply:
Why do you think it will be faster? `.some` returns on first truthy condition evaluation. It is also more expressive in this case as it reads better.

Reply:
You're totally right. Misplaced my head there for some reason :sweat_smile: 
:+1: